ui/mirage/handlers/clients.js

```diff
@@ -63,41 +93,20 @@ function generateNamespaceBlock(idx = 0, isLowerCounts = false, ns) {
     counts: {},
     mounts: {},
   };
-  const mounts = [];
 
-  Array.from(Array(5)).forEach((mount, index) => {
-    const [secretSyncs] = arrayOfCounts(randomBetween(min, max), 1);
-    mounts.push({
-      mount_path: `kvv2-engine-${index}`,
-      counts: {
-        clients: secretSyncs,
-        // TODO test with live backend to confirm entity keys are present (and 0) for kv mounts
-        entity_clients: 0,
-        non_entity_clients: 0,
-        distinct_entities: 0,
-        non_entity_tokens: 0,
-        secret_syncs: secretSyncs,
-      },
-    });
-  });
+  // * ADD NEW CLIENT TYPES HERE and spread to the mounts array below
+  const authClients = randomBetween(min, max);
+  const [non_entity_clients, entity_clients] = arrayOfCounts(authClients, 2);
```

Comment:
I wonder if instead we make this a const at the top of this page (near the first comment on how to add) so that there is no confusion?

Reply:
actually, I think I just don't understand how to update this method the right way 🤔